### ISSUE TITLE
FDSNWS event query untested

### ISSUE THREAD

**Issue by megies** [krischer/jane#59]
https://codecov.io/gh/krischer/jane/tree/master/src/jane/fdsnws

![screenshot from 2017-10-28 00-33-40](https://user-images.githubusercontent.com/1842780/32124131-03fd2e42-bb67-11e7-995e-dfebfe36d8f9.png)

**Comment by krischer** [comment]
Yes they are untested - I remember now. They have seen a more classical testing procedure ;-) But yea - should be fixed of course!

**Comment by megies** [comment]
compare also #81 which should've been discovered by tests ahead of running into problems during production..
